plugins/wasm-rust/src/plugin_wrapper.rs

```diff
@@ -257,7 +252,7 @@ where
                     }
                 }
             }
-            self.http_content.borrow().log().warn(&format!(
+            self.http_content.borrow().log().debug(&format!(
```

Comment:
well, maybe we need a log level guard because format is not zero cost, and in some condition you will build a very large string but you are not consume it

Reply:
This may need to be implemented using macros. You can create a new issue to record it

Reply:
done